File: Assignment2_4.py
#//Accept two numbers from user an display first number in second number of times

def Display(iNo1,iNo2):
  
  print("Output is:")
  for i in range (iNo2):
    print(iNo1)
    
def main():
  
  iValue1 = 0
  iValue2 = 0
  
  print("Enter the First Number:")
  iValue1 = int(input())
  
  print("Enter the Second Number:")
  iValue2 = int(input())
  
  Display(iValue1,iValue2)
  
  
if __name__=="__main__":
  main()

File: Assignment2_4.java
//Accept two numbers from user an display first number in second number of times

import java.util.*;

class Assignment2_4
{
  public static void main(String Args[])
  {
    Scanner sobj1 = new Scanner(System.in);
    Scanner sobj2 = new Scanner(System.in);

    System.out.println("Enter the First Number:");
    int iNo1 = sobj1.nextInt();

    System.out.println("Enter the Second Number:");
    int iNo2 = sobj1.nextInt();

    Demo dobj = new Demo();
    dobj.Display(iNo1, iNo2);
    sobj1.close();
    sobj2.close();
  }

}

class Demo
{
  public int iValue1;
  public int iValue2;

  Demo()
  {
    iValue1 = 0;
    iValue2 = 0;
  }

  void Display(int iValue1,int iValue2)
  {
    System.out.println("Output is:");
    for(int iCnt = 0;iCnt < iValue2;iCnt++)
    {
      System.out.println(iValue1);
    }
  }
}